src/BoundaryConditions/boundary_condition.jl

```diff
@@ -32,23 +32,24 @@ Adapt.adapt_structure(to, b::BoundaryCondition{C, A}) where {C<:BCType, A<:Abstr
 const BC   = BoundaryCondition
 const FBC  = BoundaryCondition{<:Flux}
 const PBC  = BoundaryCondition{<:Periodic}
-const NPBC = BoundaryCondition{<:NoPenetration}
+const NFBC = BoundaryCondition{<:NormalFlow}
 const VBC  = BoundaryCondition{<:Value}
 const GBC  = BoundaryCondition{<:Gradient}
-const NFBC = BoundaryCondition{Flux, Nothing}
+const ZFBC = BoundaryCondition{Flux, Nothing} # "zero" flux
```

Comment:
Got confused thinking we now had both no flux and zero flux BCs but this current setup seems good since we don't use N for "No" anymore.

Reply:
Agreed, it's a bit annoying! Fortunately, these are just internal abbreviations.